### DISCUSSION TITLE
npm run build XD

### DISCUSSION THREAD

**Discussion by Neikon** [FirefoxCSS-Store/FirefoxCSS-Store.github.io#21] [Show and tell]
![imagen](https://user-images.githubusercontent.com/1765319/106681266-740bf900-65c0-11eb-8edf-a59ba585b06b.png)
I did it XD, I needed to say it. 

I had to install a lot of things but it works now

**Comment by BeyondMagic** [comment]
That's great! The power went out in my city, so I can't commit anything (I'm using the cellphone and data mobile). Send a commit with the changes, please.

**Reply by Neikon** [reply]
Done

**Comment by unseen-ninja** [comment]
Nice! I tend to just assume that everyone is running Node/npm on their system anyway. I'm glad you got it to work tho! Now you can build all dem sources. :'D
